convert2rhel/unit_tests/subscription_test.py

```diff
@@ -172,6 +172,13 @@ def test_get_registration_cmd(self):
     def test_attach_subscription_available(self):
         self.assertEqual(subscription.attach_subscription(), True)
 
+    @unit_tests.mock(subscription, "get_avail_subs", GetAvailSubsMocked())
+    @unit_tests.mock(utils, "let_user_choose_item", LetUserChooseItemMocked())
+    @unit_tests.mock(utils, "run_subprocess", RunSubprocessMocked())
+    @unit_tests.mock(tool_opts, "activation_key", "dummy_activate_key")
+    def test_attach_subscription_available_with_activation_key(self):
+        self.assertEqual(subscription.attach_subscription(), True)
```

Comment:
Once info log has been added in the other file, mock that out and make sure it was called